services/core/java/com/android/server/pm/PackageManagerService.java

```diff
@@ -4010,13 +4010,8 @@ public void systemReady() {
         ContentObserver co = new ContentObserver(mHandler) {
```

Comment:
I think removing this ContentObserver entirely and adding
```
                for (int userId : UserManagerService.getInstance().getUserIds()) {
                    mWebInstantAppsDisabled.put(userId, true);
                }
```
in its place would be a better solution.